Formula/p/proteinortho.rb

```diff
@@ -16,19 +16,36 @@ class Proteinortho < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "bde04add25030aed32aac06465de994ca04e4410181a53deac825858df3df18c"
   end
 
+  depends_on "gcc" => :build
+  depends_on "libomp" => :build
```

Comment:
The audit suggest this is also a runtime dependency

Reply:
Thank you, will give it a try in the next commit.
The compiler is weird, on my local machine everything is fine (brew reinstall --build-from-source ...) on my mac-arm, but when uploading, it never finds the omp dylib again...

Reply:
Thats it, thank you !!

still gcc is included, but only used on linux, hope this is fine